packages.ini

```diff
@@ -0,0 +1,44 @@
+# This file contains a list of package dependencies.
+[common]
+icecast2 = buster, bionic
+postgresql = buster, bionic
+postgresql-client = buster, bionic
+rabbitmq-server = buster, bionic
+
+curl = buster, bionic
+
+# See https://github.com/Libretime/libretime/commits/71ce3082c5e0611f77eb423a15b4b36267ebfda4/installer/lib
+unzip = buster, bionic
+zip = buster, bionic
+
+[legacy]
+apache2 = buster, bionic
+php-apcu = buster, bionic
+php-pear = buster, bionic
+# Bionic
+libapache2-mod-php7.2 = bionic
+php-bcmath = bionic
+php-mbstring = bionic
+php7.2 = bionic
+php7.2-curl = bionic
+php7.2-gd = bionic
+php7.2-pgsql = bionic
+# Buster
+libapache2-mod-php7.3 = buster
+php-amqplib = buster
+php7.3 = buster
+php7.3-bcmath = buster
+php7.3-curl = buster
+php7.3-dev = buster
+php7.3-gd = buster
+php7.3-mbstring = buster
+php7.3-pgsql = buster
+
+[installer]
+lsb-release = buster, bionic
+xmlstarlet = buster, bionic
+
+[unknown]
+# See https://github.com/LibreTime/libretime/commit/bbcd4fbe2ee05cdb2e3a62ad2b7549157a1cd950#diff-0991ed0aaae32d2b0c9723cd9127f95bc19d176a5fd668d9b331f487136318fe
+systemd-sysv = buster
```

Comment:
Considering we now ship systemd unit files and don't need the sysv init scripts to run on systemd, we should be able to remove these

Reply:
Ok, wasn't sure what the purpose of this package was, didn't really looked it up. I'll remove it.

Reply:
What about the `sysvinit-utils` package for bionic ? I a not sure I understand it's purpose.

Is it related to sysV support ?

Reply:
As far as I know, it is and can be dropped too